student.rb

```diff
@@ -0,0 +1,19 @@
+require_relative('./person')
+
+class Student < Person
+  attr_accessor :classroom
+
+  # def initialize(*args, classroom, **kwargs)
+  #   super(*args, **kwargs)
+  #   @classroom = classroom
+  # end
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻